_docs/quickstarts/forseti-security/index.md

```diff
@@ -39,63 +50,29 @@ file based on your input.
-      
-      ```bash
-      python setup_forseti.py --version 1.1.0
-      ```
-  
-     _Default: runs master branch_
+  1. Invoke the setup:
      
       ```bash
       python setup_forseti.py
       ```
       
-      _Run a certain branch:_
-      
+      To see additional configurations for the setup:
+
       ```bash
-      python setup_forseti.py --branch develop
+      python setup_forseti.py -h
       ```
 
-  1. Follow the prompts to download and install the
-  [gcloud command-line tool](https://cloud.google.com/sdk/gcloud/).
-  1. Follow the prompts to authenticate your GCP account in a web browser,
-  then return to the command-line.
-  1. Enter the organization for which you want to deploy Forseti, if prompted.
-  1. Create a new project or enter an existing project ID you want to use for
-  Forseti Security. Note that it's best to use a project dedicated to running
-  Forseti.
-  1. The setup wizard checks for valid configurations or creates a new one,
-  then it checks if billing is enabled for your project.
-      * If billing isn't enabled, follow the prompts to enable billing, then
-    return to the command-line.
-  1. Next, the setup wizard automatically enables required APIs:
-  
-  {% include docs/required_apis.md %}
-  
-  1. Create a new service account or enter an existing service account for
-  accessing GCP.
-  1. Optionally create a new service account or enter an existing service
-  account for getting GSuite groups.
-  1. Next, the setup wizard automatically assigns the required roles to the GCP service
-  account. For details about the required roles, see [Forseti Security Best Practices Guide]({% link _docs/guides/best-practices.md %})
-  1. Enter a name for your bucket and Cloud SQL instance.
-  1. Setup wizard will create a Deployment Manager template based on your input
-  as well as ask whether you want to create the deployment.
-  1. Optionally create the bucket and a Cloud SQL instance. Only do this if you 
-  do not plan to create a deployment via Deployment Manager.
-  1. If you set up a service account to retrieve GSuite groups, follow the
-  post-setup instructions to enable your GSuite service account for domain-wide delegation.
-
-Forseti Security is now set up for your project and you can enable modules
-to use for your resources.
+  1. Setup will infer the necessary information to install Forseti. You will be 
+     prompted to enter a SendGrid API key, which is optional. (More information 
+     on setting up  [email notifications]({% link _docs/howto/configure/email-notification.md %}))
+     
+  1. You may be prompted to enter an ssh passphrase for Compute Engine. This will only 
+     happen if you've previously used Cloud Shell to ssh to a Compute Engine instance and
+     set a passphrase for ssh. The Forseti setup uses scp to copy the auto-generated G Suite 
+     service account key to the Forseti GCE instance.
+
+  1. After successfully running the setup and Deployment Manager, you should 
```

Comment:
This really should be a required step, but most people who've run through setup (whether the wizard or manually) had trouble with the groups enablement due to 1) need to find the G Suite superadmin to handle the G Suite part for them, 2) they got stuck trying to figure out the G Suite/GCP UI (our screenshots should help now, but we haven't had any new deployments after updating the docs)

The setup wizard can't automate the groups enablement, so that's why we have the separate link.